Description: USB backend quirk rule for Xerox Phaser 3124 (and maybe others)
Author: Jiri Popelka <jpopelka@redhat.com>
Origin: Redhat
Bugs-Upstream: http://cups.org/str.php?L4217
Last-Update: 2012-11-06

--- a/backend/usb-libusb.c
+++ b/backend/usb-libusb.c
@@ -172,6 +172,8 @@
 	{ 0x067b, 0x2305, USBLP_QUIRK_BIDIR |
 			  USBLP_QUIRK_NO_REATTACH |
 	                  USBLP_QUIRK_RESET },
+	{ 0x0924, 0x3ce9, USBLP_QUIRK_NO_REATTACH }, /* Xerox Phaser 3124
+			  https://bugzilla.redhat.com/show_bug.cgi?id=867392 */
 	/* Prolific Technology, Inc. PL2305 Parallel Port
 	   (USB -> Parallel adapter), https://bugs.launchpad.net/bugs/987485 */
 	{ 0x04e8, 0x0000, USBLP_QUIRK_RESET }, /* All Samsung devices,
